lib/v2/screens/profile_screens/guardians/guardians_tabs/guardians_screen.dart

```diff
@@ -54,12 +71,141 @@ class GuardiansScreen extends StatelessWidget {
                         title: const Text("Key Guardians"),
                         centerTitle: true,
                       ),
-                      body: TabBarView(
-                        children: [
-                          MyGuardiansTab(),
-                          MyGuardiansTab(),
-                        ],
-                      )));
+                      body: state.pageState == PageState.loading
+                          ? const FullPageLoadingIndicator()
+                          : const TabBarView(
+                              children: [
+                                MyGuardiansTab(guardianType: GuardianType.myGuardian),
+                                MyGuardiansTab(guardianType: GuardianType.imGuardian),
+                              ],
+                            )));
             })));
   }
 }
+
+void _showRecoveryStartedBottomSheet(BuildContext context, GuardianModel guardian) {
```

Comment:
we should probably get designs for bottom sheets, and perhaps have a base bottom sheet like we do with dialogs.